code/modules/loadout/categories/shoes.dm

```diff
@@ -0,0 +1,69 @@
+/// Shoe Slot Items (Deletes overrided items)
+/datum/loadout_category/shoes
+	category_name = "Shoes"
+	category_ui_icon = FA_ICON_SHOE_PRINTS
+	type_to_generate = /datum/loadout_item/shoes
+	tab_order = 1
+
+/datum/loadout_item/shoes
+	abstract_type = /datum/loadout_item/shoes
+
+/datum/loadout_item/shoes/insert_path_into_outfit(datum/outfit/outfit, mob/living/carbon/human/equipper, visuals_only = FALSE)
+	outfit.shoes = item_path
+
+/datum/loadout_item/shoes/sneakers
+	name = "Sneakers (Colorable)"
+	item_path = /obj/item/clothing/shoes/sneakers
+
+/datum/loadout_item/shoes/sandals_laced
+	name = "Sandals (Laced)"
+	item_path = /obj/item/clothing/shoes/sandal/laced
+
+/datum/loadout_item/shoes/sandals_cook_laced
+	name = "Sandals (Cook, Laced)"
+	item_path = /obj/item/clothing/shoes/cookflops/laced
+
+/datum/loadout_item/shoes/laceup
+	name = "Shoes (Laceup)"
+	item_path = /obj/item/clothing/shoes/laceup
+
+/datum/loadout_item/shoes/kindle_kicks
+	name = "Kindle Kicks"
+	item_path = /obj/item/clothing/shoes/kindle_kicks
+
+/datum/loadout_item/shoes/cowboy_brown
+	name = "Shoes (Cowboy, Brown)"
```

Comment:
This one applies for all the types of Boots - it's a bit pedantic I know, but it feels weird to select an item labelled 'shoes' and get something explicitly called something different.
Most loadout items should probably be the same as their item's name (with added notes for some, like variants or recolorable ones)

```suggestion
	name = "Boots (Cowboy, Brown)"
```